<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RenderingDialog</class>
 <widget class="QDialog" name="RenderingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>330</width>
    <height>550</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>330</width>
    <height>550</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>330</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Rendering Options</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>311</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>AntiAlias</string>
   </property>
   <widget class="QRadioButton" name="antiAliasOff">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>22</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>Off</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="antiAliasOn">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>22</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>On</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="antiAliasHigh">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>22</y>
      <width>151</width>
      <height>24</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>On (High Quality)</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>70</y>
     <width>311</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Component Text Size</string>
   </property>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>22</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Text size for part Name/Value labels</string>
    </property>
    <property name="minimum">
     <number>5</number>
    </property>
    <property name="maximum">
     <number>11</number>
    </property>
    <property name="value">
     <number>9</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>22</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Name/Value</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>22</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Text size for pin labels</string>
    </property>
    <property name="minimum">
     <number>5</number>
    </property>
    <property name="maximum">
     <number>11</number>
    </property>
    <property name="value">
     <number>9</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>22</y>
      <width>71</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Pin labels</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>311</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Target Board Dimensions</string>
   </property>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>291</width>
      <height>24</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="toolTip">
     <string>Draw area outside the target board in a darker shade</string>
    </property>
    <property name="text">
     <string>Shade off-board area</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>22</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Width</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_Height">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>52</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Height of target board</string>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>999</number>
    </property>
    <property name="value">
     <number>10</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_Width">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>22</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Width of target board</string>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>999</number>
    </property>
    <property name="value">
     <number>9</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>52</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Height</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_units_3">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>22</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>x100 mil</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_units_4">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>52</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>x100 mil</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_widths">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>311</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Track Widths (mil)</string>
   </property>
   <widget class="QSpinBox" name="holeWidth">
    <property name="geometry">
     <rect>
      <x>163</x>
      <y>40</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Hole width (1 mil = 0.001 inches)</string>
    </property>
    <property name="minimum">
     <number>20</number>
    </property>
    <property name="maximum">
     <number>40</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>26</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="padWidth">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Pad width (1 mil = 0.001 inches)</string>
    </property>
    <property name="minimum">
     <number>50</number>
    </property>
    <property name="maximum">
     <number>98</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>90</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_pad">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>18</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Pad</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_track">
    <property name="geometry">
     <rect>
      <x>87</x>
      <y>18</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Track</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_hole">
    <property name="geometry">
     <rect>
      <x>163</x>
      <y>18</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Hole</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="trackWidth">
    <property name="geometry">
     <rect>
      <x>87</x>
      <y>40</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Track width (1 mil = 0.001 inches)</string>
    </property>
    <property name="minimum">
     <number>12</number>
    </property>
    <property name="maximum">
     <number>50</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>46</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="gapWidth">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>40</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Track width (1 mil = 0.001 inches)</string>
    </property>
    <property name="minimum">
     <number>5</number>
    </property>
    <property name="maximum">
     <number>30</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_gap">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>18</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Gap</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_info">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>291</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <italic>true</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Info:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_pcb">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>359</y>
     <width>311</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>PCB Parameters (mil)</string>
   </property>
   <widget class="QLabel" name="label_mask">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>141</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Solder Mask Margin:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_silk">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>141</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Silkscreen Line Width:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="maskWidth">
    <property name="geometry">
     <rect>
      <x>163</x>
      <y>30</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Solder mask margin (1 mil = 0.001 inches)</string>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>4</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="silkWidth">
    <property name="geometry">
     <rect>
      <x>163</x>
      <y>60</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Silkscreen line width (1 mil = 0.001 inches)</string>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>7</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_edge">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>141</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Board Edge Margin:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="edgeWidth">
    <property name="geometry">
     <rect>
      <x>163</x>
      <y>90</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Board edge margin (1 mil = 0.001 inches)</string>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>50</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>20</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_viapad">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>141</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Via Pad Width:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_viahole">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>141</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Via Hole Width:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="viapadWidth">
    <property name="geometry">
     <rect>
      <x>163</x>
      <y>120</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Board edge margin (1 mil = 0.001 inches)</string>
    </property>
    <property name="minimum">
     <number>50</number>
    </property>
    <property name="maximum">
     <number>80</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>50</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="viaholeWidth">
    <property name="geometry">
     <rect>
      <x>163</x>
      <y>150</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Board edge margin (1 mil = 0.001 inches)</string>
    </property>
    <property name="minimum">
     <number>20</number>
    </property>
    <property name="maximum">
     <number>40</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>25</number>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
